support/support.go

```diff
@@ -329,6 +333,28 @@ func AgentInfo(ctx context.Context, client *codersdk.Client, log slog.Logger, ag
 		if !conn.AwaitReachable(ctx) {
 			log.Error(ctx, "timed out waiting for agent")
 		} else {
+			eg.Go(func() error {
+				mux := http.NewServeMux()
+				mux.HandleFunc("/", conn.MagicsockServeHTTPDebug)
+				req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://localhost/", nil)
+				if err != nil {
+					return xerrors.Errorf("create request: %w", err)
+				}
+				rr := httptest.NewRecorder()
+				mux.ServeHTTP(rr, req)
```

Comment:
Any reason you can't just call `conn.MagisockServeHTTPDebug(rr, req)` instead of doing it via mux?

Reply:
Completely forgot that was an option! :D